### ISSUE TITLE
Does not support async datasource

### ISSUE THREAD

**Issue by bbddmm** [imjuni/erdia#51]
https://github.com/typeorm/typeorm/issues/8914
TypeORM has supported async datasource now, but erdia doesn't.

```
 ERROR  Given data source file must contain export of a DataSource instance                                
 ERROR  Error: Given data source file must contain export of a DataSource instance            
```

**Comment by bbddmm** [comment]
https://github.com/imjuni/erdia/blob/2f3549a09c1b0278275c443a4e4f149a4b8d85d9/src/typeorm/getDataSource.ts#L37C75-L37C75

The line in question may require an `await` expression. I can confirm that adding `await` fixes the bug.

```
    if (InstanceChecker.isDataSource(await dataSourceFileExports[fileExport])) {
```

**Comment by imjuni** [comment]
fixed `3.4.0`
